src/features/problem/ui/Tabs/Discussions/Vote.tsx

```
@@ -0,0 +1,62 @@
+    if (iconType === voteStatus) {
+      setVoteStatus('NONE');
+      mutateAsync({ voteType: 'NONE' });
+    }
+    if (iconType !== voteStatus) {
+      setVoteStatus(iconType);
+      mutateAsync({ voteType: iconType });
+    }
+  };
+
+  useEffect(() => {
+    if (data && data.result) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)